tests/test_rechunk.py

```diff
@@ -88,6 +90,7 @@ def test_rechunk_group(tmp_path):
     delayed = api.rechunk(
         group, target_chunks, max_mem, target_store, temp_store=temp_store
     )
+    assert isinstance(delayed, api.Rechunked)
```

Comment:
A few lines below we call `dask.compute(delayed)`, which appears to do the right thing. But should we call `execute()` instead?

Reply:
Will take a look (probably tomorrow). I'd recommend execute, but they'll be the same if the value in the dask graph for `delayed.dask[delayed.key]` is the target array.

Reply:
I'm still thinking through this. If we return a zarr.Array here I'll need to revisit https://github.com/dask/dask/issues/5105#issuecomment-625282112 and https://github.com/dask/dask/pull/6203. Just won't have time today, or probably this week.

Anyway I would recommend using `.execute()` for now. They're equivalent as far as doing the rechunking goes, it's just that `.execute()` returns the `target` array, which may be more useful.